docs/source/nodes_and_pipelines/modular_pipelines.md

```diff
@@ -1,156 +1,66 @@
-# Modular pipelines
+# Reuse pipelines with namespaces
```

Comment:
This `modular_pipelines.md` could be `namespaces.md`, and then `modular_pipelines.md` should be for the last "how-to" sections of `pipeline_introduction.md` (see review comment)